pkg/utils/utils.go

```diff
@@ -70,10 +83,20 @@ func RetrieveVcConfigSecret(params map[string]interface{}, logger logrus.FieldLo
 			parts := strings.Split(line, "=")
 			key := strings.TrimSpace(parts[0])
 			value := strings.TrimSpace(parts[1])
-			params[key] = value[1 : len(value)-1]
+			// Skip the quotes in the value if present
+			if len(value) >= 2 && value[0] == '"' && value[len(value)-1] == '"' {
```

Comment:
It should be fine in the cases of vanilla k8s and TKG. 

But, generally, what if it is single-quoted or back-quoted?

Reply:
Do we need to support it? Seems like single quote strings are allowed, but we do not have to escape any special characters.